integration-tests/load/vrfv2/vrfv2_test.go

```diff
@@ -115,6 +115,14 @@ func TestVRFV2Performance(t *testing.T) {
 			require.NoError(t, err)
 			consumers, err = vrfv2_actions.DeployVRFV2Consumers(env.ContractDeployer, coordinator, 1)
 			require.NoError(t, err)
+			err = env.EVMClient.WaitForEvents()
+			if err != nil {
```

Comment:
Do we need this if statement when using require.NoError() below?

Reply:
ahh, copy/paste issue. thanks for the catch